src/main/java/frc/robot/controls/Driver.java

```diff
@@ -59,10 +62,12 @@ public Driver(Robot robot)
         // bind button objects to physical buttons
         lowSpeed = new JoystickButton(throttle, LOW_SPEED);
         highSpeed = new JoystickButton(throttle, HIGH_SPEED);
+        aim = new JoystickButton(steer, AIM);
 
         // bind buttons to commands
         lowSpeed.whenPressed(new LowSpeed(robot));
         highSpeed.whenPressed(new HighSpeed(robot));
+        aim.whileHeld(new VisionAim(robot));
```

Comment:
Vision aim is called when the aim button is pressed, but where do you plan on calling `AutoVisionShoot` command to calculate the shooter speed, is that going to be inside vision aim?

Reply:
`AutoVisionShoot` is being called in `Operator.java` when button 9 is pressed.